### DISCUSSION TITLE
Ayuda con error 13 permisos denegados (Kata 10)

### DISCUSSION THREAD

**Discussion by RobertoVergaraD** [LaunchX-InnovaccionVirtual/CursoIntroPython#299] [Q&A] [answered]
Alguien me puede ayudar con este error:

![image](https://user-images.githubusercontent.com/99287441/154596664-1d59ec4b-1669-491f-8ef1-deaa21535ab3.png)

Me marca permiso denegado y no me permite seguir con los demás ejercicios.

**Comment by doguedogue** [comment, marked as answer]
En realidad ese error está bien también, para cuando creas el directorio.
Si creas el archivo ese error no aparecería.

Es algo extraño que Windows capture ese error como de permisos.

En Linux, WSL o una terminal aparece el error de que es un directorio como lo dice la Kata.

Checa esa pregunta se ha hecho varias veces aquí y tiene números respuestas.

**Reply by RobertoVergaraD** [reply]
Muchas gracias por tu respuesta, busque en otras preguntas pero no vi específicamente el error 13 ó si había solución para él, pero les vuelvo a dar otra checada.

Saludos!
